docker-compose.yml

```diff
@@ -19,6 +19,15 @@ services:
       context: .
       dockerfile: Dockerfile.test
     command: ["./scripts/test_e2e.sh"]
+  prometheus:
+    image: prom/prometheus:latest
+    ports:
+      - 9090:9090 
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
```

Comment:
Should I create another volume for storage?

Reply:
yes, I think you should create another volume for storage in here